nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/resources/skin/views/doc/emptyDocAdapter.ftl

```diff
@@ -0,0 +1,47 @@
+<#include "views/doc/macros.ftl"/>
+<@extends src="baseapi.ftl">
+<@block name="apis">
+  {
+    "path": "/@emptyWithDefault",
```

Comment:
Note that these won't return any initialized properties without the `properties: *` header.

Reply:
You're right, will update the doc.